utils/utils.go

```diff
@@ -138,3 +161,21 @@ func ReadRemoteFile(url string) (string, error) {
 
 	return buf.String(), nil
 }
+
+// ReadLocalFile takes in the location of a local file
+// in the format `file://location/of/file` and returns
+// the content of the file if the path is valid and no
+// error occurs
+func ReadLocalFile(location string) (string, error) {
+	// remove the protocol prefix
+	location = strings.TrimPrefix(location, "file://")
+
+	// Need to support variable file locations hence
+	// #nosec
+	data, err := ioutil.ReadFile(location)
+	if err != nil {
+		return "", err
```

Comment:
Should we be using a specific MeshKit error code here? I'm not sure. Just asking.

Reply:
yes